### DISCUSSION TITLE
java/javaSE%E5%9F%BA%E7%A1%80/01_Java%E5%85%A5%E9%97%A8%E4%B8%8E%E5%BC%80%E5%8F%91%E7%8E%AF%E5%A2%83%E6%90%AD%E5%BB%BA

### DISCUSSION THREAD

**Discussion by giscus** [hjc1732/technical_detective_display#2] [Announcements]
# java/javaSE%E5%9F%BA%E7%A1%80/01_Java%E5%85%A5%E9%97%A8%E4%B8%8E%E5%BC%80%E5%8F%91%E7%8E%AF%E5%A2%83%E6%90%AD%E5%BB%BA

Java入门与开发环境搭建 一、计算机基础知识 1.1、软件开发 什么是软件？ 软件（Software）是一系列按照特定顺序组织的计算机数据和指令的集合 软件分类： 系统软件（Windows、Linux、Android、IOS） 应用软件（QQ、迅雷、微信...） 什么是软件开发？ 软件开发是根据用户要求建造出软件系统或者系统中的软件部分的过程； 软件...

http://localhost:8080/java/javaSE%E5%9F%BA%E7%A1%80/01_Java%E5%85%A5%E9%97%A8%E4%B8%8E%E5%BC%80%E5%8F%91%E7%8E%AF%E5%A2%83%E6%90%AD%E5%BB%BA.html

**Comment by hjc1732** [comment]
好好学一学

**Comment by hjc1732** [comment]
基础
